### DISCUSSION TITLE
Why is the path to finding a package like this

### DISCUSSION THREAD

**Discussion by myNameIsDu** [eslint/eslint#14611] [Q&A] [answered]
I encountered this error when using @babel/eslint-parser
`Parsing error: Cannot find module '@babel/preset-env'

Require stack:
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@babel/core/lib/config/files/plugins.js
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@babel/core/lib/config/files/index.js
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@babel/core/lib/index.js
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@babel/eslint-parser/lib/worker/babel-core.cjs
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@babel/eslint-parser/lib/worker/index.cjs
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@babel/eslint-parser/lib/client.cjs
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@babel/eslint-parser/lib/analyze-scope.cjs
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@babel/eslint-parser/lib/index.cjs
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@eslint/eslintrc/lib/config-array-factory.js
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/@eslint/eslintrc/lib/index.js
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/eslint/lib/cli-engine/cli-engine.js
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/eslint/lib/cli-engine/index.js
- /Users/du/Documents/my/my-blogger/blo-react/node_modules/eslint/lib/api.js
- /Users/du/.vscode/extensions/dbaeumer.vscode-eslint-2.1.20/server/out/eslintServer.jseslint`

This is my configuration
`    parser: '@babel/eslint-parser',
    parserOptions: {
        requireConfigFile: false,
        babelOptions: {
            presets: ['@babel/preset-react'],
            presets: [
                '@babel/preset-env',
                [
                    '@babel/preset-react',
                    {
                        runtime: 'automatic',
                    },
                ],
            ],
        },
    },`

**Comment by aladdin-add** [comment, marked as answer]
`@babel/eslint-parser` requires babel related deps installed.
```bash
npm i @babel/preset-env @babel/preset-react -D
```

**Reply by myNameIsDu** [reply]
![image](https://user-images.githubusercontent.com/53305259/119371077-9c8f3e80-bce8-11eb-8179-0edede161f7b.png)

They do exist, and I think there's a problem with the path I'm looking for

![image](https://user-images.githubusercontent.com/53305259/119371260-d06a6400-bce8-11eb-916a-af8aa56bc70e.png)

**Reply by aladdin-add** [reply]
how did you get the error, command line?

please make sure the command line works(e.g. `npx eslint .`)

**Reply by myNameIsDu** [reply]
this is an IDE error, the command line is fine
